content/es/docs/sitemap.md

```diff
@@ -0,0 +1,114 @@
+  if(tag) {
+    tag = $.trim(tag);
+    for (i=0;i<tagName.length;i++) {
+      querystringTag = tagName[i] + "=";
+      if (tag.indexOf(querystringTag) > -1)
+      {
+        console.log("in mainLog: querystringTag of " + querystringTag + " matches tag of " + tag);
+        tag = tag.replace(querystringTag,"");
+        selectDropDown(tagName[i],tag);
+        topicsFilter(tagName[i],tag,"output");
+      }
+    }
+  } else {
+    currentTopics = metadata.pages;
+  }
+  renderTable(currentTopics,"output");
+
+}
+function populateDropdowns()
+{
+  // Keeping mainLogic() brief by functionalizing the initialization of the
+  // drop-down filter boxes
+
+  for(i=0;i<metadata.pages.length;i++)
+  {
+    var metadataArrays = [metadata.pages[i].cr,metadata.pages[i].or,metadata.pages[i].mr];
+    for(j=0;j<metadataArrays.length;j++)
+    {
+      if (metadataArrays[j]) {
+        for (k=0;k<metadataArrays[j].length;k++) {
+          if (typeof storedTagsArrays[j] == 'undefined') storedTagsArrays[j] = new Array();
+          storedTagsArrays[j][metadataArrays[j][k][tagName[j]]] = true;
+          // ^ conceptList[metadata.pages[i].cr[k].concept] = true; (if rolling through concepts)
+          // ^ conceptList['container'] = true; (ultimate result)
+          // ^ objectList[metadata.pages[i].or[k].object] = true; (if rolling through objects)
+          // ^ objectList['restartPolicy'] = true; (ultimate result)
+        }
+      }
+    }
+  }
+  var output = new Array();
+  for(i=0;i<tagName.length;i++)
+  {
+    // Phew! All tags in conceptList, objectList, and commandList!
+    // Loop through them and populate those drop-downs through html() injection
+    output = [];
+    output.push("<select id='" + tagName[i] + "' onchange='dropFilter(this)'>");
+    output.push("<option>---</option>");
+    Object.keys(storedTagsArrays[i]).sort().forEach(function (key) {
+      output.push("<option>" + key + "</option>");
+    });
+    output.push("</select>")
+    $(dropDowns[i]).html(output.join(""));
+  }
+  dropDownsPopulated = true;
+}
+function dropFilter(srcobj)
+{
+  // process the change of a drop-down value
+  // the ID of the drop down is either command, object, or concept
+  // these exact values are what topicsFilter() expects, plus a filter val
+  // which we get from .text() of :selected
+  console.log("dropFilter:" + $(srcobj).attr('id') + ":" + $(srcobj).find(":selected").text());
+  topicsFilter($(srcobj).attr('id').replace("#",""),$(srcobj).find(":selected").text(),"output");
+  for(i=0;i<tagName.length;i++)
+  {
+    if($(srcobj).attr('id')!=tagName[i]) selectDropDown(tagName[i],"---");
+  }
+}
+function selectDropDown(type,tag)
+{
+  // change drop-down selection w/o filtering
+  $("#" + type).val(tag);
+}
+</script>
+<style>
+#filters select{
+  font-size: 14px;
+  border: 1px #000 solid;
+}
+#filters {
+  padding-top: 20px;
+}
+</style>
+
+Click tags or use the drop downs to filter. Click table headers to sort or reverse sort.
```

Comment:
Me parece que este parrafo debe ser traducido

Reply:
Fixed in https://github.com/kubernetes/website/pull/13543/commits/ec0c232821f565d0f719caed01d78f02e6400421

Reply:
Fixed in https://github.com/kubernetes/website/commit/21c69932e27b921cb657a09d7e7ff17e2a8938f0